SteamKit2/SteamKit2/Steam/CDNClient.cs

```diff
@@ -497,7 +497,7 @@ async Task<byte[]> DoRawCommandAsync( Server server, string command, string? arg
 
                     if ( !response.IsSuccessStatusCode )
                     {
-                        throw new SteamKitWebRequestException( $"Response status code does not indicate success: {response.StatusCode} ({response.ReasonPhrase}).", response );
```

Comment:
The more-correct fix here would be `{response.StatusCode:D}`, which calls into `Enum.ToString(string)` as per https://docs.microsoft.com/en-us/dotnet/standard/base-types/enumeration-format-strings?view=netframework-4.8

Reply:
I wasn't familiar with the "D" format string. I've edited both files.